common/config/rush/.pnpmfile.cjs

```diff
@@ -28,6 +28,9 @@ function readPackage(pkg) {
     ].includes(pkg.name)
   ) {
     pkg.dependencies["@itwin/core-bentley"] = "workspace:*";
+    if (pkg.name === "@itwin/browser-authorization" || pkg.name === "@itwin/electron-authorization") {
+      pkg.dependencies["@itwin/core-common"] = "workspace:*";
```

Comment:
@ben-polinsky makes me wonder if our deps in these pkgs are right

Reply:
I was thinking the same. The justification for the direct dependency was that we're using types from core-common in our public api. 

Practically, would anyone auth packages without core-common? Perhaps some, but it's only a slight inconvenience to install.

Reply:
very unlikely, might as well roll your own auth. resolved comment to prevent blocking any future rush update fulls